src/luxonis_ml/data/dataset.py

```diff
@@ -218,141 +129,96 @@ def __init__(
         if not isinstance(self.bucket_storage, BucketStorage):
             raise Exception(f"Must use a valid BucketStorage!")
 
+        self.online = self.team_id != "offline"
```

Comment:
Should we make offline mode more clear compared to setting it as a team_id.

Reply:
Yeah I thought about this as well. Maybe setting an environment variable and/or a `config.json` for `data` would make sense? Like `LUXONISML_ONLINE=true`?